src/lib/components/controls/MappingForm.svelte

```diff
@@ -42,6 +46,25 @@
 	const statusOptions = ['planned', 'implemented', 'verified'];
 
 	const isValid = $derived(formData.justification.trim().length > 0);
+	
+	const cciOptions = $derived(
+		(cci ?? '')
+			.split(';')
+			.map((s) => s.trim())
+			.filter(Boolean)
+	);
+
+	
+	let selectedCCIs = $state<string[]>(
+		(formData.cci ?? '')
```

Comment:
This appears to throw a svelte warning:
```
[1] 3:41:09 PM [vite-plugin-svelte] src/lib/components/controls/MappingForm.svelte:60:3 This reference only captures the initial value of `formData`. Did you mean to reference it inside a derived instead?
[1] https://svelte.dev/e/state_referenced_locally
```

Although doesn't seem to affect functionality.

Reply:
Yes it is. thank you for that callout